code/modules/antagonists/changeling/powers/mutations.dm

```diff
@@ -144,6 +144,7 @@
 	req_human = 1
 	weapon_type = /obj/item/melee/arm_blade
 	weapon_name_simple = "blade"
+	resistance_flags = ACID_PROOF
 
 /obj/item/melee/arm_blade
```

Comment:
resistance flags  variable should be placed inside this instead of its current spot